src/components/Garden/Metrics.js

```diff
@@ -289,67 +296,47 @@ function TokenPrice({
       <div
         css={`
           display: flex;
-          justify-content: space-between;
+          align-items: center;
+          ${textStyle('title4')};
+          margin-bottom: ${0.5 * GU}px;
         `}
       >
-        <p
-          css={`
-            ${textStyle('title4')};
-            margin-bottom: ${0.5 * GU}px;
-          `}
-        >
-          {token.symbol} Price{' '}
-          {oracleMode && (
-            <span
-              css={`
-                ${textStyle('body4')}
-              `}
-            >
-              (Oracle)
-            </span>
```

Comment:
Let´s perhaps leave this smaller as it can cause overflow for the rest of the metrics. When we have the dedicated metrics , balances page this shouldn´t be a problem :D
![Screen Shot 2021-11-28 at 22 44 31](https://user-images.githubusercontent.com/22663232/143796539-ca95add4-66d5-47a5-bcee-c003a003ddc3.png)